code-submissions-clustering-plugin/build.gradle.kts

```diff
@@ -47,18 +49,23 @@ tasks {
             if (project.hasProperty("saveClusters")) {
                 add("--saveClusters")
             }
+            if (project.hasProperty("clusteringResult")) {
```

Comment:
Please, add examples into the main README file how we can run these tasks and examples of the results